db/migrations/000020_add_helmfile_and_fc_develop_ref_fields.up.sql

```diff
@@ -0,0 +1,17 @@
+alter table v2_chart_releases
+    add if not exists firecloud_develop_ref text;
+
+alter table v2_changesets
+    add if not exists from_firecloud_develop_ref text;
+
+alter table v2_changesets
+    add if not exists to_firecloud_develop_ref text;
+
+alter table v2_clusters
+    add if not exists helmfile_ref text;
+
+alter table v2_environments
+    add if not exists helmfile_ref text;
+
+alter table v2_clusters
+    add if not exists location text;
```

Comment:
Gorm and the migration files are inconsistent here. The gorm structs say that some of these columns need not null constraints and defaults, but the migration files don't uphold that

Reply:
Ah good catch, I added the gorm stuff as I went, and forgot to go back and update this